app/models/user.rb

```diff
@@ -0,0 +1,14 @@
+class User < ApplicationRecord
+  before_save :downcase_email
+
+  validates :name, presence: true, length: {maximum: 50}
+
+  validates :email, uniqueness: true, length: {maximum: 255}
```

Comment:
```suggestion
  validates :name, presence: true, length: {maximum: 50}
  validates :email, uniqueness: true, length: {maximum: 255}
```
các gía trị 255 20 thì khai báo vào setting hì
em tìm hiểu về gem config để làm nà